Docs 36 labels multiseries (#12772)
metabase/metabase: docs/users-guide/05-visualizing-results.md

# Visualizing results

While tables are useful for looking up information or finding specific numbers, it's usually easier to see trends and make sense of data using charts.

In Metabase, an answer to a question can be visualized in a number of ways:

- [Numbers](#numbers)
- [Trend](#trends)
- [Progress bar](#progress-bars)
- [Gauge](#gauges)
- [Table](#tables)
- [Line chart](#line-bar-and-area-charts)
- [Bar chart](#line-bar-and-area-charts)

> - [Combo chart: line + bar chart](#line-plus-bar-charts)

Reviewer: We just call these "Combo" charts in the product now.

```suggestion
- [Combo chart](#line-plus-bar-charts)
```

### Line, bar, and area charts

**Line charts** are best for displaying the trend of a number over time, especially when you have lots of x-axis values. Bar charts are great for displaying a number grouped by a category (e.g., the number of users you have by country). Bar charts can also be useful for showing a number over time if you have a smaller number of x-axis values (like orders per month this year).

![Bar chart](images/visualizations/bar.png)

**Area charts** are useful when comparing the proportions of two metrics over time. Both bar and area charts can be stacked.

![Stacked area chart](images/visualizations/area.png)

**Trend lines**

**Trend lines** are another useful option for line, area, bar, and scatter charts. If you have a question where you're grouping by a time field, open up the visualization settings and turn the `Show trend line` toggle on to display a trend line. Metabase will choose the best type of line to fit to the trend of your series. Trend lines will even work if you have multiple metrics selected in your summary. But trend lines won't work if you have any groupings beyond the one time field.

![Trend lines](images/visualizations/trend-lines.png)

> ### Combo charts: Line + Bar charts <a name="line-plus-bar-charts"></a>

Reviewer: ```suggestion
### Combo charts <a name="line-plus-bar-charts"></a>
```

> Also called **combo charts**, the line + bar chart lets you combine bars and lines (or areas) on the same chart.

Reviewer: ```suggestion
Combo charts let you combine bars and lines (or areas) on the same chart.
```

![Line + bar](images/visualizations/combo-chart.png)

Metabase will pick one of your series to display as a line, and another to display as a bar by default. Open up the visualization settings to change which series are lines, bars, or areas, as well as to change per-series settings like colors. Click the down arrow icon on the right of a series to see additional options:

![Line + bar](images/visualizations/combo-chart-settings.png)

> To use a Line + Bar chart, you'll either need to have two or more metrics selected in the Summarize By section of your question, with one or two grouping columns, like this:

Reviewer: ```suggestion
To use a Combo chart you'll either need to have two or more metrics selected in the Summarize By section of your question, with one or two grouping columns, like this:
```

**Axes**

There are three main ways to configure axes:

- **Change the scale for your axes**. If you're looking at a time series chart, your x-axis can use a time series scale or an ordinal one. Your y-axis can use a linear, power, or logarithmic scale.
- **Hide or show the tick marks on your axes**. You can also choose to rotate the tick marks on the x-axis to help them fit better.
- **Edit the range of your y-axis**. Metabase sets an automatic range by default, but you can toggle that off and input a custom minimum and maximum value for the y-axis if you'd like.

**Labels**

Here's where you can choose to hide the **label** for your x- or y-axis. You can also customize the text for your axes labels here.

### Scatterplots and bubble charts

**Scatterplots** are useful for visualizing the correlation between two variables, like comparing the age of your users vs. how many dollars they've spent on your products. To use a scatterplot, you'll need to ask a question that results in two numeric columns, like `Count of Orders grouped by Customer Age`. Alternatively, you can use a raw data table and select the two numeric fields you want to use in the chart options.

> If you have a third numeric field, you can also create a **bubble chart**. Select the Scatter visualization, then open up the chart settings and select a field in the `bubble size` dropdown. This field will be used to determine the size of each bubble on your chart. For example, you could use a field that contains the total dollar amount for each x-y pair — i.e., larger bubbles for larger total dollar amounts spent on orders.

Reviewer: ```suggestion
If you have a third numeric field, you can also create a **bubble chart**. Select the Scatter visualization, then open up the chart settings and select a field in the **bubble size** dropdown. This field will be used to determine the size of each bubble on your chart. For example, you could use a field that contains the total dollar amount for each x-y pair — i.e., larger bubbles for larger total dollar amounts spent on orders.
```
